weis/frequency/raft_wrapper.py

```diff
@@ -252,7 +252,27 @@ def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
             for ii in range(s_ballast.shape[0]):
                 iball = np.where(s_ballast[ii,0] >= s_grid)[0][-1]
                 rho_fill[iball] = rho_ballast[ii]
-                l_fill[iball] = inputs[f"member{k}:height"] * h_ballast[ii] if rho_ballast[ii] > 1100.0 else var_height[k]
+                # Initial ballast fill values
```

Comment:
Does the current code for non-water ballast need to be scaled by the member height?  Units would be off otherwise? 
https://github.com/WISDEM/WEIS/blob/develop/weis/frequency/raft_wrapper.py#L255C33-L255C59
vs
https://github.com/WISDEM/WEIS/blob/phase2/weis/frequency/raft_wrapper.py#L257

Reply:
Yeah, I think this is the way.  Otherwise, RAFT complains about the fill levels. RAFT wants all its ballasts defined in the same units as the `stations`/`s_grid` provided.  Since we're giving it the stations relative to the member, the ballast fills should also be relative to the member.  I checked the OC3 and VolturnUS response in RAFT and was satisfied with the results.

Reply:
Sounds like you fixed a bug then!